src/components/donation/Steps/DonateMethods/Stripe/Stripe.tsx

```diff
@@ -0,0 +1,126 @@
+import { Elements } from "@stripe/react-stripe-js";
+import { loadStripe } from "@stripe/stripe-js";
+import { useState } from "react";
+import { FormValues, Props } from "./types";
+import { useCreateStripePaymentIntentMutation } from "services/apes";
+import { useErrorContext } from "contexts/ErrorContext";
+import KYCForm from "components/KYCForm";
+import { KYC } from "slices/donation";
+import { PUBLIC_STRIPE_KEY } from "constants/env";
+import Checkout from "./Checkout";
+import Form from "./Form";
+
+const stripePromise = loadStripe(PUBLIC_STRIPE_KEY);
+
+// Followed Stripe's custom flow docs
+// https://stripe.com/docs/payments/quickstart
+export default function Stripe(props: Props) {
```

Comment:
@misicnenad, I got error when choosing credit/debit
<img width="1392" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/ed474558-d343-4cb1-94db-129d9f899c12">

```

data
: 
{message: 'Failed to get currencies'}
status
: 
400
```

repro:
`git branch -D stripe-embed` - delete local  

`git pull `RC-v1.9.2` - fetch latest remotes  

`git checkout stripe-embed`  

`yarn && yarn start`  

`go to donate/33`

Reply:
Will be addresses as part of https://linear.app/better-giving/issue/BG-1072/fix-bug-in-fetch-currencies

@SovereignAndrey @ap-justin in the meantime to test things out:
- open [src/services/apes/apes.ts](https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/stripe-embed/src/services/apes/apes.ts)
- swap out `${country_code}` on line 57 for a hardcoded value of `RS` (this works for me), so that the URL looks like:
``url: `v2/fiat/stripe-proxy/apes/${apiEnv}/currencies/RS`,``
- reload the donation page
- verify everything works.

Reply:
Created a fix https://github.com/AngelProtocolFinance/devops/pull/464, currency loading should work now